compilator/compilator.go

```diff
@@ -249,6 +275,10 @@ func (j compileJob) Run() {
 					color.MagentaString(j.pkg.Release.Name),
 					color.MagentaString(j.pkg.Name))
 				j.doneCh <- compileResult{pkg: j.pkg, err: errWorkerAbort}
+
+				if c.MetricsPath != "" {
+					stampy.Stamp(c.MetricsPath, "fissile", waitSeriesName, "done")
```

Comment:
Sorry I didn't spot this before:
Does this mean the wait series `done` event won't happen until the other side has finished reading your compileResult? (I don't think `j.doneCh` is buffered).